qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/common/data_handling/data_presenter.py

```diff
@@ -38,25 +36,20 @@ def __init__(self, model, view):
         self.plot_added_notifier = GenericObservable()
         self.plot_removed_notifier = GenericObservable()
         self.all_plots_removed_notifier = GenericObservable()
-        self.fit_all_started_notifier = GenericObservable()
         # Observers
-        self.fit_enabled_observer = GenericObserverWithArgPassing(self.set_fit_enabled)
         self.focus_run_observer = GenericObserverWithArgPassing(self.view.set_default_files)
 
-    def set_fit_enabled(self, fit_enabled):
-        self.view.set_fit_buttons_enabled(fit_enabled)
+    def get_sorted_active_ws_list(self):
+        return self.model.get_active_ws_sorted_by_primary_log()
 
-    def fit_completed(self, fit_props):
-        if fit_props:
-            self.model.update_fit(fit_props)
+    def get_active_ws_list(self):
+        return self.model.get_active_ws_name_list()
 
-    def _start_seq_fit(self):
-        ws_name_list = self.model.get_active_ws_sorted_by_primary_log()
-        self.fit_all_started_notifier.notify_subscribers((ws_name_list, True))
+    def get_loaded_ws_list(self):
+        return self.model.get_loaded_ws_list()
 
-    def _start_serial_fit(self):
-        ws_name_list = self.model.get_active_ws_name_list()
-        self.fit_all_started_notifier.notify_subscribers((ws_name_list, False))
+    def get_log_ws_name(self):
+        return self.model.get_log_workspaces_name()
```

Comment:
See my other comment first.
I wonder whether the `get_log_workspaces_name` was badly named. It used to return a list of the individual workspaces (run_info, Temp_1, W_position etc) that were inside the workspace group called something like ENGIN-X_305761-305671_logs. 
If you need the workspace group name here I think you need to call another (new) function